Resources/Prototypes/_Harmony/Accents/word_replacements.yml

```
@@ -0,0 +1,205 @@
+    accent-irish-words-123: accent-irish-words-replace-123
+    accent-irish-words-124: accent-irish-words-replace-124
+    accent-irish-words-125: accent-irish-words-replace-125
+    accent-irish-words-126: accent-irish-words-replace-126
+    accent-irish-words-127: accent-irish-words-replace-127
+    accent-irish-words-128: accent-irish-words-replace-128
+    accent-irish-words-129: accent-irish-words-replace-129
+    accent-irish-words-130: accent-irish-words-replace-130
+    accent-irish-words-131: accent-irish-words-replace-131
+    accent-irish-words-132: accent-irish-words-replace-132
+    accent-irish-words-133: accent-irish-words-replace-133
+    accent-irish-words-134: accent-irish-words-replace-134
+    accent-irish-words-135: accent-irish-words-replace-135
+    accent-irish-words-136: accent-irish-words-replace-136
+    accent-irish-words-137: accent-irish-words-replace-137
+    accent-irish-words-138: accent-irish-words-replace-138
+    accent-irish-words-139: accent-irish-words-replace-139
+    accent-irish-words-140: accent-irish-words-replace-140
+    accent-irish-words-141: accent-irish-words-replace-141
+    accent-irish-words-142: accent-irish-words-replace-142
+    accent-irish-words-143: accent-irish-words-replace-143
+    accent-irish-words-144: accent-irish-words-replace-144
+    accent-irish-words-145: accent-irish-words-replace-145
+    accent-irish-words-146: accent-irish-words-replace-146
+    accent-irish-words-147: accent-irish-words-replace-147
+    accent-irish-words-148: accent-irish-words-replace-148
+    accent-irish-words-149: accent-irish-words-replace-149
+    accent-irish-words-150: accent-irish-words-replace-150
+    accent-irish-words-151: accent-irish-words-replace-151
+    accent-irish-words-152: accent-irish-words-replace-152
+    accent-irish-words-153: accent-irish-words-replace-153
+    accent-irish-words-154: accent-irish-words-replace-154
+    accent-irish-words-155: accent-irish-words-replace-155
+    accent-irish-words-156: accent-irish-words-replace-156
+    accent-irish-words-157: accent-irish-words-replace-157
+    accent-irish-words-158: accent-irish-words-replace-158
+    accent-irish-words-159: accent-irish-words-replace-159
+    accent-irish-words-160: accent-irish-words-replace-160
+    accent-irish-words-161: accent-irish-words-replace-161
+    accent-irish-words-162: accent-irish-words-replace-162
+    accent-irish-words-163: accent-irish-words-replace-163
+    accent-irish-words-164: accent-irish-words-replace-164
+    accent-irish-words-165: accent-irish-words-replace-165
+    accent-irish-words-165-2: accent-irish-words-replace-165
+    accent-irish-words-166: accent-irish-words-replace-166
+    accent-irish-words-167: accent-irish-words-replace-167
+    accent-irish-words-168: accent-irish-words-replace-168
+    accent-irish-words-169: accent-irish-words-replace-169
+    accent-irish-words-170: accent-irish-words-replace-170
+- type: accent
```

The accents should have a space between for better readability
```suggestion
    accent-irish-words-170: accent-irish-words-replace-170

- type: accent
```